viewer/viewer.js

```diff
@@ -8224,6 +8297,87 @@ app.put('/hunt/:id/play', [noCacheJson, logAction('hunt/:id/play'), checkCookieT
   updateHuntStatus(req, res, 'queued', 'Queued hunt item successfully', 'Error starting hunt job');
 });
 
+app.delete('/hunt/:id/users/:user', [noCacheJson, logAction('hunt/:id/users/:user'), checkCookieToken, checkPermissions(['packetSearch']), checkHuntAccess], (req, res) => {
+  if (Config.get('multiES', false)) { return res.molochError(401, 'Not supported in multies'); }
+
+  Db.get('hunts', 'hunt', req.params.id, (err, hit) => {
```

Comment:
probably should create Db.getHunt since we have all the other hunt stuff there already

Reply:
👍